### ISSUE TITLE
listPlans endpoint: setIds parameter

### ISSUE THREAD

**Issue by muratovi** [recurly/recurly-client-java#47]
**Describe the bug**

With given list of 3 valid plan ids only one Plan object is returned. 

```
        val params = QueryParams()
        params.setIds(ids)

        client.listPlans(params)
                .forEach { println(it) }
```

Method `getPlan` returns value for each given plan id:
```
client.getPlan(planId)
```

**Your Environment**

client version 3.1.0

**Comment by douglasmiller** [comment]
@muratovi Thank you very much for bringing this to our attention. 

Unfortunately, this is a bug in our openapi specification. The `ids` parameter should not be a `List`. It should be a `String` of comma separated id values.

The description in the [documentation ](https://developers.recurly.com/api/v2019-10-10/index.html#operation/list_plans) is correct, but the field type is incorrect:

```
Filter results by their IDs. Up to 200 IDs can be passed at once using commas as separators,
e.g. ids=h1at4d57xlmy,gyqgg0d3v9n1,jrsm5b4yefg6.
```

We will correct this issue asap!

**Comment by bhelx** [comment]
Thank you for reporting @muratovi 

This has been published in [3.1.1](https://github.com/recurly/recurly-client-java/releases/tag/3.1.1)
Please give it a little time to reach maven central: https://search.maven.org/artifact/com.recurly.v3/api-client
